types/relay-runtime/lib/store/RelayStoreTypes.d.ts

```diff
@@ -482,7 +482,7 @@ export interface Environment {
      * Optionally takes an owner, corresponding to the operation that
      * owns this selector (fragment).
      */
-    lookup(selector: SingularReaderSelector): Snapshot;
+    lookup(selector: SingularReaderSelector | ReaderSelector): Snapshot;
```

Comment:
@alloy should this really be `lookup(selector: ReaderSelector)` then if that `ReaderSelector` is a union of both `SingularReaderSelector` and `PluralReaderSelector`?